app/src/main/java/com/github/swent/echo/data/supabase/SupabaseDataSource.kt

```diff
@@ -0,0 +1,98 @@
+package com.github.swent.echo.data.supabase
+
+import com.github.swent.echo.data.model.Association
+import com.github.swent.echo.data.model.Event
+import com.github.swent.echo.data.model.Tag
+import com.github.swent.echo.data.model.UserProfile
+import com.github.swent.echo.data.repository.datasources.RemoteDataSource
+import com.github.swent.echo.data.supabase.entities.EventSupabase
+import com.github.swent.echo.data.supabase.entities.EventSupabaseSetter
+import com.github.swent.echo.data.supabase.entities.EventTagSupabase
+import com.github.swent.echo.data.supabase.entities.UserProfileSupabase
+import com.github.swent.echo.data.supabase.entities.UserProfileSupabaseSetter
+import com.github.swent.echo.data.supabase.entities.UserTagSupabase
+import io.github.jan.supabase.SupabaseClient
+import io.github.jan.supabase.postgrest.from
+import io.github.jan.supabase.postgrest.query.Columns
+
+class SupabaseDataSource(supabaseClient: SupabaseClient) : RemoteDataSource {
```

Comment:
You could use `(private val supabase: SupabaseClient)` instead and remove `var supabase = supabaseClient`.

Reply:
you're right... I just moved this file actually and didn't touch this part o the code, but it makes sense obviously! I'll change that.